tox.ini

```diff
@@ -43,10 +42,10 @@ deps =
     devdeps: photutils>=0.0.dev0
     devdeps: git+https://github.com/spacetelescope/synphot_refactor.git#egg=synphot
 
-    oldestdeps: numpy==1.22.*
+    oldestdeps: numpy==1.24.*
```

Comment:
While you are at it, might also want to sync this with minversion pins in pyproject.toml ?

Reply:
Bumping Python and dep versions probably needs a change log as well. Thanks!